tests/gold_tests/bigobj/bigobj.test.py

```diff
@@ -52,65 +47,88 @@
 )
 
 ts.Disk.remap_config.AddLine(
-    'map https://localhost http://localhost'
+    f'map https://localhost:{ts.Variables.ssl_port} http://localhost:{ts.Variables.port}'
+)
+ts.Disk.remap_config.AddLine(
+    f'map https://localhost:{ts.Variables.ssl_portv6} http://localhost:{ts.Variables.port}'
 )
-
-# Set up to check the output after the tests have run.
-#
-log_id = Test.Disk.File("log2.txt")
-log_id.Content = "log2.gold"
 
 # Size of object to get.  (NOTE:  If you increase this significantly you may also have to increase cache
 # capacity in tests/gold_tests/autest-size/min_cfg/storage.config.  Also, for very large objects, if
 # proxy.config.diags.debug.enabled is 1, the PUSH request will timeout and fail.)
 #
 obj_kilobytes = 10 * 1024
+obj_bytes = obj_kilobytes * 10
+header = "HTTP/1.1 200 OK\r\nContent-length: {}\r\n\r\n".format(obj_bytes)
+
+
+def create_pushfile():
+    f = open(Test.RunDirectory + "/objfile", "w")
+    f.write(header)
+    f.write("x" * obj_bytes)
```

Comment:
I think there' s some value in having a ramping pattern, rather than a constant value for all bytes in the file.  For example, if somehow ATS swapped the order of two IOBuffers.

Reply:
Seems reasonable